packages/swap/src/routes/swap.ts

```diff
@@ -176,70 +172,18 @@ router.get(
   }),
 );
 
-let client: BrokerClient | undefined;
-
+// TODO(major): remove this handler. it's replaced by tRPC
```

Comment:
any reason why we didn't remove this ? the sdk directly calls the trpc method now right?